Dockerfile

```diff
@@ -53,21 +60,24 @@ RUN addgroup -g 1001 -S nodejs
 RUN adduser -S nextjs -u 1001
 
 COPY --from=builder /app/public ./public
-
-# Automatically leverage output traces to reduce image size
-# https://nextjs.org/docs/advanced-features/output-file-tracing
 COPY --from=builder --chown=nextjs:nodejs /app/.next/standalone ./
 COPY --from=builder --chown=nextjs:nodejs /app/.next/static ./.next/static
-
 COPY --from=builder /app/package.json ./
 
 USER nextjs
 
-# Expose the port Next.js runs on
-EXPOSE $FRONTEND_SERVICE_PORT
```

Comment:
I'm curious why the switch from the variable to a hardcoded port 3000?

Reply:
Thanks for the educational response, I learned something and I support the aim of being more human-friendly.